src/node/desktop/src/core/user.ts

```diff
@@ -14,10 +14,17 @@
  */
 
 import os from 'os';
+import { getenv } from './environment';
 
 import { FilePath } from './file-path';
 
 export function userHomePath(): FilePath {
+  const user = getenv('R_USER');
+  if (user != '') 
```

Comment:
Because of Typescript, both sides are ensured to be strings but for style consistency `!==` should be used to also check type.